<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>841</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Surname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Street</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>House</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Math</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Physics</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Language</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitialize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSurname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarkMath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAMark">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStreet">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHouse">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarkPhys">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarkLang">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>290</y>
      <width>481</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnShow">
       <property name="text">
        <string>Show All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelAMark">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>406</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Mark</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelSurname">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCity">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStreet">
       <property name="text">
        <string>Street</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelHouse">
       <property name="text">
        <string>House</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>407</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelMarkMath">
       <property name="text">
        <string>Mark Math</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMarkPhys">
       <property name="text">
        <string>Mark Physics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMarkLang">
       <property name="text">
        <string>Mark Language</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
